package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -672,7 +672,7 @@ cdef class _NSGrid(object):
 
         cdef ns_int i, cellindex = -1
         cdef ns_int ncoords = coords.shape[0]
-        cdef ns_int[:] beadcounts = np.empty(self.size, dtype=np.int)
+        cdef ns_int[:] beadcounts = np.empty(self.size, dtype=int)
```

Comment:
the nsgrid tests still fail because I've not re-generated the c files, but it's not worth doing this until #3133 is fixed